src/main/java/io/xpring/xrpl/javascript/JavaScriptUtils.java

```diff
@@ -49,14 +49,24 @@ public boolean isValidAddress(String address) {
    * @return A new X-Address if inputs were valid, otherwise null.
    * @see <a href="https://xrpaddress.info/">https://xrpaddress.info/</a>
    */
+  @SuppressWarnings("checkstyle:LocalVariableName")
   public String encodeXAddress(ClassicAddress classicAddress) {
     Objects.requireNonNull(classicAddress);
 
     Value encodeXAddressFunction = javaScriptUtils.getMember("encodeXAddress");
-    Value result = classicAddress.tag().isPresent()
-        ? encodeXAddressFunction.execute(classicAddress.address(), classicAddress.tag().get(), classicAddress.isTest())
-        : encodeXAddressFunction.execute(classicAddress.address(), classicAddress.isTest());
-    return result.asString();
+
+    if (classicAddress.tag().isPresent()) {
+      Value xAddress = encodeXAddressFunction.execute(
+          classicAddress.address(),
+          classicAddress.tag().get(),
+          classicAddress.isTest()
+      );
+      return xAddress.asString();
+    } else {
+      Value undefined = JavaScriptLoader.getContext().eval("js", "undefined");
+      Value xAddress = encodeXAddressFunction.execute(classicAddress.address(), undefined, classicAddress.isTest());
```

Comment:
Got it. So the fix here was that you were passing `isTest()` to the `tag` parameter of this function, and now you pass `undefined`?

Reply:
Correct. The `isTest` parameter got added in some update to Xpring-JS and I just appended rather than thinking about what was actually happening.